orchestration/src/test/java/com/sap/ai/sdk/orchestration/OrchestrationUnitTest.java

```diff
@@ -380,17 +380,4 @@ void testErrorHandling() {
 
     softly.assertAll();
   }
-
-  @Test
-  void testEmptyChoicesResponse() {
```

Comment:
(Comment)

I understand it doesn't make sense to cover edge-cases with unit tests.
I just feel defensive about removing test code.

Reply:
Yeah, in this case I was worried that people later look at this test case and incorrectly conclude this is a valid scenario that might happen, because it is tested